chandra_aca/aca_image.py

```diff
@@ -194,7 +194,7 @@ def _adjust_item(self, item):
 
         out_rc = [None, None]  # New [row0, col0]
 
-        if isinstance(item, (int, np.int)):
+        if isinstance(item, (int, np.int_)):
```

Comment:
Should be `int`.

Reply:
No. This is a check on the input type, which can by the np type.

Reply:
Perhaps a better way would be to use np.atleast_1d.

Reply:
I added a line of test to show a failure if I remove this `np_int_`. This kind of input is expected.

Reply:
Good catch. This is now a bug fix instead of just the mindless transformation to get rid of the deprecation warning (i.e. the original code was incorrect as well).

Instead of `np.int_` I think it should be `np.integer`. See the top image in https://axil.github.io/numpy-data-types.html. And `np.int_` is a little funny:
```
In [12]: issubclass(np.byte, np.int_)                                                                                 
Out[12]: False

In [13]: issubclass(np.byte, np.integer)                                                                              
Out[13]: True
```
So generally speaking we should be doing `isinstance(value, (int, np.integer))`. This is how a generalized int is checked in astropy.

Reply:
Just added a change to use np.integer.